server/routes/varyingLicences/handlers/viewVariation.ts

```diff
@@ -21,6 +23,28 @@ export default class ViewVariationRoutes {
       LicenceStatus.VARIATION_REJECTED,
     ].includes(<LicenceStatus>licence.statusCode)
 
+    if (
+      licence.statusCode === LicenceStatus.VARIATION_SUBMITTED ||
+      licence.statusCode === LicenceStatus.VARIATION_REJECTED ||
+      licence.statusCode === LicenceStatus.VARIATION_APPROVED
+    ) {
+      // If submitted or rejected view the variation summary and approval conversation
+      const [conversation, conditionComparison] = await Promise.all([
+        this.licenceService.getApprovalConversation(licence, user),
+        this.licenceService.compareVariationToOriginal(licence, user),
+      ])
+
+      return res.render(`pages/vary/viewSubmitted`, {
+        conversation,
+        conditionComparison,
+        callToActions: { shouldShowVaryButton, shouldShowPrintButton, shouldShowEditAndDiscardButton },
+      })
+    }
+
+    // Viewing an ACTIVE licence - show conditions and prompt to vary
+    const expandedLicenceConditions = expandAdditionalConditions(licence.additionalLicenceConditions)
+    const expandedPssConditions = expandAdditionalConditions(licence.additionalPssConditions)
+
     return res.render('pages/vary/viewVariation', {
```

Comment:
This alters which form is displayed first depending on the status of the variation - so the conversation is seen up-front for any submitted, referred or approved variation.